be/src/olap/tablet_manager.cpp

```diff
@@ -1249,10 +1332,11 @@ void TabletManager::try_delete_unused_tablet_path(DataDir* data_dir, TTabletId t
         return;
     }
 
-    if (shard.tablets_under_clone.count(tablet_id) > 0) {
-        LOG(INFO) << "tablet is under clone, skip delete the path " << schema_hash_path;
+    bool succ = register_transition_tablet(tablet_id, "path gc");
```

Comment:
at line 1330 is not right.  even if tablet meta is exists,  this data dir may be unused  because its path's shard id is different with exists tablets.